DOL.WHD.Section14c.DataAccess/DOL.WHD.Section14c.DataAccess.csproj

```diff
@@ -323,8 +323,8 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="MigrationsDB2\201709281739066_InitialDB2.resx">
-      <DependentUpon>201709281739066_InitialDB2.cs</DependentUpon>
```

Comment:
I'd like some commentary around this change to the migrations.  It looks like just a file rename, and I'm curious why that's necessary.  I'm also curious how DevArt tracks which migrations have been applied - if it's based on filenames, then changing migration script names will break migrations on existing installations.  A little clarity here would be good prior to merging.

Reply:
I didn't make any changes to the database. The file was regenerated for testing purpose. Feel free to ignore this change.

Reply:
Does that mean that Visual Studio created a new resource file and renamed the `201709281739066_InitialDB2.cs` file to `201710181418022_InitialDB2.cs`?  The `csproj` file being updated doesn't worry me, Visual Studio changes those and solution files in strange and unpredictable ways all the time.  😛 

I just want to make sure I understand why those `...InitialDB2.cs` and resource files were changed so when it happens in the future, I know what I'm seeing and how to evaluate it.  🙂

Reply:
@mgwalker Your observation is correct and was needed to embed the license file so Jenkins server and web servers do not need to install the devart product :)

Reply:
I do not believe the embedded resource needs to be tied to migrations though.

Reply:
I'm going to guess for now that the resource file contains something about the model that changed because of DevArt (maybe a new table to track migrations or something?).  That'll satisfy me for now.  😛